CactbotOverlay/CactbotOverlayConfigPanel.cs

```diff
@@ -69,28 +69,47 @@ public partial class CactbotOverlayConfigPanel : UserControl {
 
     private void SetupFileWatcher()
     {
+      System.Collections.Generic.List<string> pathsList = new System.Collections.Generic.List<string>();
+      
+      // Overlay Dir
       if (this.config.Url == "")
         return;
-      var path = System.IO.Path.GetDirectoryName(config.Url);
-      path = System.Text.RegularExpressions.Regex.Replace(path, @"file:[\\\/]+", "");
-      if (!System.IO.Directory.Exists(path))
-      {
-        this.overlay.LogError("Directory does not exist!" + path);
-        return;
+      var overlayPath = System.IO.Path.GetDirectoryName(config.Url);
+      overlayPath = System.Text.RegularExpressions.Regex.Replace(overlayPath, @"file:[\\\/]+", "");
+      pathsList.Add(overlayPath);
+
+      // User Dir
+      string basePath = System.Text.RegularExpressions.Regex.Match(overlayPath, @"(.*cactbot)").Groups[1].ToString();
+      string userPath = System.IO.Path.Combine(basePath, "user");
```

Comment:
The user directory can exist in a number of places.  It can be relative to the html, relative to the dll, or specified explicitly as an option.